front/app/containers/Admin/dashboard/reports/ProjectReport.tsx

```diff
@@ -312,33 +317,61 @@ const ProjectReport = memo(
                 />
               </>
             </GraphsContainer>
-          </Section>
-        )}
-      </>
+          )}
+          {participationMethods.includes('poll') ? (
+            isTimelineProject ? (
+              !isNilOrError(phases) &&
+              phases.map(
+                (phase) =>
+                  phase.attributes.participation_method === 'poll' && (
+                    <PollReport
+                      participationContextType="phase"
+                      participationContextId={phase.id}
+                      participationContextTitle={localize(
+                        phase.attributes.title_multiloc
+                      )}
+                    />
+                  )
+              )
+            ) : (
+              <PollReport
+                participationContextType="project"
+                participationContextId={project.id}
+                participationContextTitle={localize(
+                  project.attributes.title_multiloc
+                )}
+              />
+            )
+          ) : null}
+        </Section>
+      </ThemeProvider>
     );
   }
 );
 
-const ProjectReportWithHoc = injectIntl(ProjectReport);
+const ProjectReportWithHoc = withRouter(injectIntl(ProjectReport));
 
-const Data = adopt<DataProps, InputProps>({
-  phases: ({ project, render }) => (
-    <GetPhases projectId={project.id}>{render}</GetPhases>
+const Data = adopt<DataProps, WithRouterProps>({
+  phases: ({ params, render }) => (
+    <GetPhases projectId={params.projectId}>{render}</GetPhases>
   ),
-  mostVotedIdeas: ({ project, render }) => (
+  mostVotedIdeas: ({ params, render }) => (
     <GetIdeas
       pageNumber={1}
       pageSize={5}
       sort="popular"
       type="paginated"
-      projectIds={[project.id]}
+      projectIds={[params.projectId]}
     >
       {render}
     </GetIdeas>
   ),
+  project: ({ params, render }) => (
+    <GetProject projectId={params.projectId}>{render}</GetProject>
+  ),
 });
 
-export default (inputProps: InputProps) => (
+export default (inputProps: WithRouterProps) => (
```

Comment:
If there will be WithRouterProps here, you should wrap with WithRouter here, instead of in the ProjectReportWithHoc var. 

```
export default withRouter((inputProps: WithRouterProps) => (
...
```